help_to_heat/frontdoor/schemas.py

```diff
@@ -805,7 +805,7 @@ class SessionSchema(Schema):
     uprn = fields.String()
     rrn = fields.String()
     epc_details = fields.Dict()
-    address_and_rrn_details = fields.List(fields.Dict)
+    address_and_lmk_details = fields.List(fields.Dict)
```

Comment:
this will rename a field in session data, though since this is only used temporarily & not for routing this should not have much impact on in-flight users